packages/core/src/utils/applyOpacityToColor.test.ts

```diff
@@ -0,0 +1,48 @@
+import { applyOpacityToColor } from "./applyOpacityToColor";
+import { describe, expect, it } from "vitest";
+
+describe("applyOpacityToColor", () => {
+  const cases: [string, number, string][] = [
+    // RGB
```

Comment:
What happens if you use the default browse colours? `teal`, `pink`, `black`, `white`, `aliceblue` etc. Would they still work? Worth adding them to the test case to make it explicit.

Reply:
Great point! Named colors just don't work with big opacity right now, but the list of named colors is small enough that we could just create a name → hex map for those and get it working! I'll add that in here today.